test/visibility/submodule-in-file/errors/WrongModule1.good

```diff
@@ -0,0 +1 @@
+error: Cannot include module from module with different file name
```

Comment:
Is the reason for this error to enforce a specific style / avoid confusion about whether the search directory should be named after the file or the module?  Is this something we might consider relaxing later if people bristled at it?  (For example, I could imagine having `OtherModule.chpl` and then creating clones of it like `OtherModule-hack.chpl`, `OtherModule-v2.chpl`, etc. and not wanting to create new subdirectories for each clone.  Bryant's answer would be to use symbolic links, but that feels clumsy to me, and I don't see it as inherently problematic to base the directory name on the actual module name rather than the filename.  It's a rule to learn, but not the worst thing in the world.  (Personally, I'd just permit this from the get-go, but I'm also OK with starting strict and relaxing down the road if others agree with me).

(this also suggests another place we could look for submodules if we wanted to search in multiple places as mentioned in a previous comment).

Reply:
> Is the reason for this error to enforce a specific style / avoid confusion about whether the search directory should be named after the file or the module?

Yes

>  Is this something we might consider relaxing later if people bristled at it?

Yes

>  I'm also OK with starting strict and relaxing down the road if others agree with me

Let's do that :)

Reply:
Afterthought: As a practical application of this relaxed mode, I think a number of your tests could've each had their own distinct filename, a shared module name, and then used a set of common submodules in a single directory rather than needing to replicate similar logic / structures multiple times for multiple distinct tests.

Reply:
Yep. One thing I am curious about is if we could base the directory-to-search-in on the filename AND the module name (i.e. search in both). Or something. But since making such things errors now is not backwards-breaking, I'd like to be more strict for the moment.